src/ray/common/status.cc

```diff
@@ -145,6 +145,11 @@ std::string Status::ToString() const {
   result += ": ";
   result += state_->msg;
 
+  if (IsRpcError()) {
+    result += " rpc_code: ";
+    result += std::to_string(state_->rpc_code);
```

Comment:
nit (no merge blocker): prefer not to use `std::to_string` and use `absl::StrFormat`, because there's a global lock inside of the implementation, and I met performance issues before.

Reference hackernews link: https://news.ycombinator.com/item?id=30472423
Reference official doc: https://en.cppreference.com/w/cpp/string/basic_string/to_string

Reply:
^ The above is nit, non merge blocker; if prompt message is needed we need to use stringstream (or expose a mutable function from `Status`)

Another nit (also non merge blocker) I have is, it's better to `std::move(os).str()`, so after we upgrade to C++20 we get performance gain with no code change.